deployments/nimbus/Chart.yaml

```diff
@@ -1,24 +1,32 @@
 apiVersion: v2
 name: nimbus
-description: A Helm chart for Nimbus
-
-# A chart can be either an 'application' or a 'library' chart.
-#
-# Application charts are a collection of templates that can be packaged into versioned archives
-# to be deployed.
-#
-# Library charts provide useful utilities or functions for the chart developer. They're included as
-# a dependency of application charts to inject those utilities and functions into the rendering
-# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+description: A Helm chart for Nimbus operator.
 type: application
-
+kubeVersion: ">= 1.25"
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.1
 
+version: "0.1.2"
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
+
 appVersion: "0.1.1"
+
+dependencies:
```

Comment:
Please add k8tls adapter also

Reply:
Since, all helm charts are released only as part of our stable releases. If we add the k8tls chart now, users installing nimbus from local charts will encounter errors until the next stable release is published. This is because the k8tls chart is not included in the current stable release.